app/scripts/directives/metas/metas-actividades.js

```diff
@@ -263,6 +263,7 @@ angular
                 }
               }
             })
+            console.log(fuentes);
```

Comment:
Comentar o eliminar llamados a console.log
```suggestion
            // console.log(fuentes);
```
```suggestion
```